src/js/player.js

```diff
@@ -276,6 +258,44 @@ class Player extends Component {
     let techComponent = Component.getComponent(techName);
     this.tech = new techComponent(this, techOptions);
 
+    // Listen to every HTML5 events and trigger them back on the player for the plugins
+    this.on(this.tech, 'loadstart', this.onTechLoadStart);
+    this.on(this.tech, 'waiting', this.onTechWaiting);
+    this.on(this.tech, 'canplay', this.onTechCanPlay);
+    this.on(this.tech, 'canplaythrough', this.onTechCanPlayThrough);
+    this.on(this.tech, 'playing', this.onTechPlaying);
+    this.on(this.tech, 'ended', this.onTechEnded);
+    this.on(this.tech, 'seeking', this.onTechSeeking);
+    this.on(this.tech, 'seeked', this.onTechSeeked);
+    this.on(this.tech, 'ended', this.onTechEnded);
+    this.on(this.tech, 'play', this.onTechPlay);
+    this.on(this.tech, 'firstplay', this.onTechFirstPlay);
+    this.on(this.tech, 'pause', this.onTechPause);
+    this.on(this.tech, 'progress', this.onTechProgress);
+    this.on(this.tech, 'durationchange', this.onTechDurationChange);
+    this.on(this.tech, 'fullscreenchange', this.onTechFullscreenChange);
```

Comment:
This fullscreenchange listener could still be removed, now that you've added it to the player.